src/modules/contacts/pages/ContactOverviewPage.vue

```diff
@@ -17,34 +17,196 @@
       <AppHeading>{{ t('contactOverview.information') }}</AppHeading>
+      <AppInfoList>
+        <AppInfoListItem
+          :name="'Preferred channel'"
+          :value="contact.profile.preferredContact"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.email')"
+          :value="contact.email"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.phone')"
+          :value="contact.profile.telephone"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.deliveryOptIn')"
+          :value="contact.profile.deliveryOptIn"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.newsletter')"
+          :value="contact.profile.newsletterStatus"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.groups')"
+          :value="contact.profile.newsletterGroups.join(',')"
+        />
+      </AppInfoList>
+    </div>
+
+    <div>
+      <AppHeading>{{ t('contactOverview.contribution') }}</AppHeading>
+      <AppInfoList>
+        <AppInfoListItem
+          :name="t('contacts.data.amount')"
+          :value="
+            contact.contributionAmount
+              ? n(contact.contributionAmount, 'currency')
+              : '-'
+          "
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.period')"
+          :value="contact.contributionPeriod"
+        />
+        <AppInfoListItem
+          v-if="contact.contribution.type === ContributionType.GoCardless"
+          :name="t('contacts.data.payingFee')"
+          :value="contact.contribution.payFee"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.contributionType')"
+          :value="contact.contribution.type"
+        />
+      </AppInfoList>
+    </div>
+    <div class="row-span-3">
+      <AppHeading>{{ t('contactOverview.about') }}</AppHeading>
+      <p class="mb-5">
+        {{ t('contactOverview.annotation.copy.begin') }}
+        <b>{{ t('contactOverview.annotation.copy.bold') }}</b>
+        {{ t('contactOverview.annotation.copy.end') }}
+      </p>
+
+      <form @submit.prevent="handleFormSubmit">
+        <AppInput
+          v-model="contactAnnotations.notes"
+          input-type="text"
+          :label="'Notes'"
+          class="mb-4"
+        />
+        <AppTextArea
+          v-model="contactAnnotations.description"
+          input-type="text"
+          :label="'Description'"
+        />
+        <AppButton
+          type="submit"
+          variant="link"
+          class="mt-4"
+          :loading="loading"
+          >{{ t('form.saveChanges') }}</AppButton
+        >
+      </form>
+    </div>
+
+    <div>
+      <AppHeading>{{ t('contactOverview.roles') }}</AppHeading>
+      <AppInfoList v-for="role in contact.roles" :key="role.role">
+        <AppInfoListItem :name="t(`common.role.${role.role}`)">
+          <!-- @TODO: make component -->
+          <span class="inline-block w-2 h-2 bg-success rounded-full"></span>
```

Comment:
Make this into a little component which takes a colour as an attribute

Reply:
Done: https://github.com/beabee-communityrm/beabee-frontend/pull/104/commits/5e4f54269287cd70ffe74ac20a25b3c328a4b585